tox.ini

```diff
@@ -1,16 +1,16 @@
 [tox]
-envlist = flake8, pydocstyle, py27, py35
+envlist = py36
 
 
-[testenv:py27]
-commands = 
-    python -m pytest --doctest-modules scholia
-    python -m pytest tests
-deps=
-    pytest
-    -rrequirements.txt
+#[testenv:py27]
+#commands = 
+#    python -m pytest --doctest-modules scholia
+#    python -m pytest tests
+#deps=
+#    pytest
+#    -rrequirements.txt
 
-[testenv:py35]
```

Comment:
if you just the magical header `[testenv]` then it uses whatever version of python on which tox is installed